client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -162,15 +161,24 @@ button {
  	 * Step Container
  	 */
 	.step-container {
-		--color-accent: #117ac9;
-		--color-accent-60: #0e64a5;
 
 		.form-fieldset {
 			label {
 				text-transform: none;
 			}
 		}
 	}
+
+	// While we don't standardize all Calypso interfaces, we need to override this for onboarding flows #79851
+	.components-button.is-primary {
+		&[disabled],
+		&:disabled,
+		&.disabled {
+			color: #fff;
```

Comment:
Your screenshots looked quite good with this changed to grey https://github.com/Automattic/wp-calypso/pull/79851#issuecomment-1650627544, did we decide to just match production?

Reply:
https://github.com/Automattic/wp-calypso/pull/79851#issuecomment-1651136523